apps/mobile-alphabet/src/app/Components/alphabet-card-detail.presenter.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useEffect, useState } from 'react';
+import { Button, Image, Text, View } from 'react-native';
+import config from './Config.json';
+import { AlphabetData } from './Menu';
+
+export function AlphabetCardDetailPresenter() {
+    const [alphabetData, setAlphabetData] = useState<AlphabetData | null>(null);
+
+    // Sequence numbers are indexed starting at 1
+    const [selectedLetterSequenceNumber, setSelectedLetterSequenceNumber] = useState<number>(1);
+
+    useEffect(() => {
```

Comment:
Why did you bring the state into the presenter? The point of the presenter is that it takes one `AlphabetCard` as props and presents it. You should leave all the stuff around alphabet data and selected sequence number in the parent component.